js/apps/admin-ui/src/realm-settings/TokensTab.tsx

```diff
@@ -85,6 +88,25 @@ export const RealmSettingsTokensTab = ({
     defaultValue: false,
   });
 
+  const signedMetadataEnabled = useWatch({
+    control,
+    name: convertAttributeNameToForm(
+      "attributes.oid4vci.signed_metadata.enabled",
+    ),
+    defaultValue: realm.attributes?.["oid4vci.signed_metadata.enabled"],
+  });
+  const encryptionRequired = useWatch({
+    control,
+    name: convertAttributeNameToForm("attributes.oid4vci.encryption.required"),
```

Comment:
An empty line would be welcome.

```suggestion
    defaultValue: realm.attributes?.["oid4vci.signed_metadata.enabled"],
  });
  
  const encryptionRequired = useWatch({
    control,
    name: convertAttributeNameToForm("attributes.oid4vci.encryption.required"),
```